### DISCUSSION TITLE
Can another project use the mkdocs-material home page layout/assets

### DISCUSSION THREAD

**Discussion by srinisubramanian** [squidfunk/mkdocs-material#2841] [Q&A] [answered]
Not sure if this is already answered, could not find it.

Can another mkdocs project with material theme, copy and use the home page and assets as present in the mkdocs-material home page here: https://squidfunk.github.io/mkdocs-material/ ?

**Comment by squidfunk** [comment, marked as answer]
In general, you can use everything, as it is licensed under MIT. However, I'd kindly ask you not to use the home page illustration, which is custom-made for Material for MkDocs.

**Reply by srinisubramanian** [reply]
Thanks for the reply.  So except for the illustration (https://squidfunk.github.io/mkdocs-material/assets/images/illustration.png) the other artifacts (html/svg etc) should be fine is what you are indicating?

Thanks for a wonderful addition to mkdocs.  The home page is so neat and clean that I wanted to make use of it.
